pkg/v1/model/model.go

```diff
@@ -635,6 +635,38 @@ func (s *Sequential) CloneLearnablesTo(to *Sequential) error {
 	return nil
 }
 
+// SetLearnables sets learnables to model
+func (s *Sequential) SetLearnables(desired g.Nodes) error {
+	destination := s.trainChain.Learnables()
+	if len(desired) != len(destination) {
+		return fmt.Errorf("models must be identical to clone learnables")
```

Comment:
nit: `cannot set learnables: number of desired nodes not equal to number of nodes in model`

Reply:
Oh boy, you're right, my bad 😅

Reply:
thanks!